build-system/compile/compile.js

```diff
@@ -120,13 +123,11 @@ function cleanupBuildDir() {
  * Generates a list of source files based on various factors.
  * TODO(wg-infra, wg-performance): Clean up unnecessary files.
  *
- * @param {string[]|string} entryModuleFilenames
- * @param {string} outputDir
- * @param {string} outputFilename
+ * @param {string[]} entryModuleFilenames
  * @param {!OptionsDef} options
  * @return {!Array<string>}
  */
-function getSrcs(entryModuleFilenames, outputDir, outputFilename, options) {
+function getSrcs(entryModuleFilenames, options) {
```

Comment:
More proof that type-checking does help.

Reply:
Yeah, I think it's really highlighted by the `TODO` on line 190 which shows you thought there needed to be clean up here but it wasn't clear that two of the parameters were not being used.